### ISSUE TITLE
VPN Settings (gear) icon is missing its tooltip

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#22636]
## Description 

VPN Settings (gear) icon is missing its tooltip

## Steps to Reproduce

   1. install `1.40.16`
   2. launch Brave
   3. set `brave://flags/#brave-vpn` to `Enabled`
   4. click on `Relaunch`
   5. click on the `VPN` icon in the browser toolbar
   6. hover over the `Settings` (gear) icon in the doorhanger panel

## Actual result:

No tooltip

<img width="974" alt="Screen Shot 2022-04-29 at 11 45 18 PM" src="https://user-images.githubusercontent.com/387249/166095344-6d216a0c-8d79-48c4-b764-562c2d317973.png">

## Expected result:

Perhaps `VPN Settings` or just `Settings`, as a tooltip?  We need to do what's best for accessibility (if `Settings` would be too ambiguous with a screenreader, for instance).

## Reproduces how often: 

100%

## Brave version (brave://version info)

Brave | 1.40.16 Chromium: 101.0.4951.41 (Official Build) nightly (x86_64)
-- | --
Revision | 93c720db8323b3ec10d056025ab95c23a31997c9-refs/branch-heads/4951@{#904}
OS | macOS Version 12.4 (Build 21F5063e)

cc @aguscruiz @bsclifton @nullhook

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

Brave | 1.51.58 Chromium: 112.0.5615.29 (Official Build) nightly (x86_64)
-- | --
Revision | 8a59304c62e066962363d2fd8edd8e7b459c42bd-refs/branch-heads/5615@{#497}
OS | macOS Version 13.3 (Build 22E252)

### Steps:
1. installed `1.51.58`
2. launched Brave
3. restarted Brave to enable `VPN` via Griffin
4. loaded `account.brave.com`
5. logged in using `stephen-prod@mailinator.com`
6. linked account via `Refresh Brave VPN` button
7. clicked on the `VPN` button
8. hovered over the icon

#### Confirmed the tooltip text of `Settings` when hovering over the `Gears`/`Settings` icon


<img width="1199" alt="Screenshot 2023-03-23 at 5 13 08 PM" src="https://user-images.githubusercontent.com/387249/227391697-dc884a91-a48a-4eb1-b80c-66d1e5a2c24d.png">

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

Brave | 1.51.58 Chromium: 112.0.5615.29 (Official Build) nightly (64-bit)
-- | --
Revision | 8a59304c62e066962363d2fd8edd8e7b459c42bd-refs/branch-heads/5615@{#497}
OS | Windows 10 Version 22H2 (Build 19045.2728)

### Steps:

1. installed `1.51.58`
2. launched Brave
3. restarted Brave to enable `Brave VPN` via Griffin
4. loaded `account.brave.com`
5. logged in using `stephen-prod@mailinator.com`
6. linked account via `Refresh Brave VPN` button
7. clicked on the` VPN` button
8. hovered over the Settings icon


#### Confirmed the tooltip text of `Settings` when hovering over the `Gears/Settings` icon

![Capture](https://user-images.githubusercontent.com/387249/227394334-d8f1f96b-884d-47f2-b27d-805a8149ea35.PNG)
